src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -2,21 +2,24 @@
 
 import com.revrobotics.CANSparkMax;
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
+import edu.wpi.first.wpilibj.DoubleSolenoid;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
 import edu.wpi.first.wpilibj.PneumaticsModuleType;
 import edu.wpi.first.wpilibj.Solenoid;
 import edu.wpi.first.wpilibj.motorcontrol.MotorControllerGroup;
 import edu.wpi.first.wpilibj2.command.SubsystemBase;
 import frc.robot.Constants;
 
 /**
- * Climber subsytem, defines all climber motor and pneumatics
+ * Climber subsystem, defines all climber motor and pneumatics
  */
 public class Climber extends SubsystemBase {
 
     private final Solenoid outsideClimberSolenoid = new Solenoid(Constants.Pneumatics.pcm1,
-        PneumaticsModuleType.CTREPCM, Constants.Pneumatics.Climber.outsideChannel);
-    private final Solenoid insideClimberSolenoid = new Solenoid(Constants.Pneumatics.pcm2,
-        PneumaticsModuleType.CTREPCM, Constants.Pneumatics.Climber.insideChannel);
+        PneumaticsModuleType.CTREPCM, Constants.Pneumatics.climberOutsideChannel);
+    private final DoubleSolenoid insideClimberSolenoid =
+        new DoubleSolenoid(Constants.Pneumatics.pcm2, PneumaticsModuleType.CTREPCM,
+            Constants.Pneumatics.climberInsideChannel, Constants.Pneumatics.climberInsideChannel2);
 
     private final CANSparkMax outsideClimberMotor1 =
```

Comment:
Is there ever a time when we will need to run both inside and outside climbers at the same time? With different commands. We may need to split this into separate subsystems later or make new commands that include both sets of motors.